src/content/ContentLoaded.module.scss

```diff
@@ -1,6 +1,16 @@
 @import './clinical-lowlight-theme/ContentLoaded.module';
 @import './orion-fusion-theme/ContentLoaded.module';
 
+:local {
+  .scroll {
+    height: 100%;
+    overflow: auto;
+    -webkit-overflow-scrolling: touch;
+    position: relative;
+    width: 100%;
+  }
+}
+
 :local(.markdown) {
```

Comment:
I remember using this syntax before, but I don't remember what this does

Reply:
That class specifically is local (hashed), but everything under it is global (not hashed).